taxi/views.py

```diff
@@ -57,7 +87,26 @@ class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
-    queryset = Car.objects.all().select_related("manufacturer")
+
+    def get_context_data(
+            self,
+            *,
+            object_list: list = None,
```

Comment:
add additional typehint for list inner objects like list[ANY_TYPE]
use typehint like: <ANY_TYPE> | None = None